docs/modules/operation/pages/deep-dive/service-assurance/scheduled-outages.adoc

```diff
@@ -0,0 +1,45 @@
+
+= Scheduled Outages
+
+Scheduled Outages allow for suppressing notifications and pausing Status Polling, Threshold Checking, and Data Collection in {page-component-title}.
+{page-component-title} supports many configurable options for when scheduled outages need to occur.
+
+Scheduled Outages is located under Admin -> Service Monitoring -> Configure Scheduled Outages
+
+== Components of a Scheduled Outage
+A Scheduled Outage definition requires the following:
+
+. Schedule Outage Name
+. Nodes to include in the scheduled outage
+. Interfaces on a node to include in the scheduled outage
+. Outage type Specific/Daily/Weekly/Monthly
+. Date/Time the outage occurs
+. Suppression of the following
+** All Notifications
+** Status Polling
+** Threshold Checking
+** Data Collection
+
+When an outage is scheduled, any nodes and/or interfaces matching the outage definition will be suppressed.
+
+== Editing Outage
+
+To enable or configure the geocoder service, use the web-based configuration tool:
+
+. In the {page-component-title} web UI, enter a name for the Schedule Outage and click the *plus* symbol in the top-right of the page.
+. Under Node Labels, click the rectangular box, select a node to include in the scheduled outage, and click *Add*
+. Under Interfaces, click the rectangular box, select a node interface to include in the scheduled outage, and click *Add*
```

Comment:
```suggestion
. Under Interfaces, select a node interface to include in the scheduled outage and click *Add*.
```

Reply:
I had put rectangular box since it isn't quite clear that the rectangular box _**is**_ initiates the picker.
<img width="471" alt="Screenshot 2023-02-17 at 3 16 17 PM" src="https://user-images.githubusercontent.com/111781759/219815166-5ad8ead2-d250-4bfd-ad3b-ddabdbe33f26.png">
I am also not certain how the "Add with path outage dependency" works.